designate.md

```diff
@@ -0,0 +1,28 @@
+Deploying Designate
+-------------------
+
+Designate provides SUSE OpenStack Cloud, DNS as a service. It allows creating and propagating zones
+and records over the network using pools of DNS server. From an Admin's perspective there is not
+much to configure as the default for deployment are well in place.
+
+This designate barclamp relies heavily on dns barclamp and expects it to be applied without any
+failures.
+
+- designate-server role
+this role will install the designate server packages and configure mini-dns (mdns) service required
+by designate
+
+- designate-worker role
+this role configures a designate worker on the selected nodes. Designate uses the workers to distribute
+its workload
+
```

Comment:
if you are not supporting Sink, (a good idea IMHO) , should we mention Sink is not included

Reply:
yes we should atleast mention it